### ISSUE TITLE
Outbound shipment: ListView table parameters should be in the URL

### ISSUE THREAD

**Issue by joshxg** [openmsupply/openmsupply-client#160]
More of a question :)

Should the `sortBy`, `limit`, `offset` all be in the URL so they can be linked to?

**Comment by mark-prins** [comment]
yeah, probably should. Not such a high priority for me, I don't see that linking to a particular page or sort option is super helpful. it is consistent though and sets the pattern, which I think is important.

**Comment by joshxg** [comment]
Realise this could also be a slightly better ux if you're to navigate back as your search filtering etc could be persisted
